evap/staff/tests/test_importers.py

```diff
@@ -359,3 +353,14 @@ def test_import_new_participant(self):
 
         self.assertEqual(self.evaluation1.participants.count(), 2)
         self.assertEqual(set(self.evaluation1.participants.all()), set([self.participant1, self.participant2]))
+
+def create_memory_excel_file(data):
+    memory_excel_file = io.BytesIO()
+    wb = xlwt.Workbook()
+    for sheet_name, sheet_data in data.items():
+        sheet = wb.add_sheet(sheet_name)
+        for (row_data, row_num) in zip(sheet_data, range(len(sheet_data))):
```

Comment:
You can replace `zip(x, range(len(x)))` with [`enumerate(x)`](https://docs.python.org/3/library/functions.html#enumerate):
```suggestion
        for (row_num, row_data) in enumerate(sheet_data):
```

Reply:
Nice, Thanks 👍